content/developer/reference/javascript/generic_components.rst

```diff
@@ -7,6 +7,22 @@ To make it easier, the Odoo javascript framework provides a suite of generic
 components that can be reused in some common situations, such as dropdowns,
 checkboxes or datepickers. This page explains how to use these generic components.
 
+Reference List
+==============
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Technical Name
+     - Short Description
+   * - :ref:`CheckBox <generic_components/checkbox>`
```

Comment:
Tip: If an internal hyperlink target immediately precedes a heading, it's not necessary to specify the label in the `:ref:`, depending on whether you want to copy the title of the heading or not.